quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/KeycloakProcessor.java

```diff
@@ -270,24 +280,23 @@ private void configureJpaProperties(ParsedPersistenceXmlDescriptor descriptor, H
         }
     }
 
-    private void configureJpaModel(ParsedPersistenceXmlDescriptor descriptor, CombinedIndexBuildItem indexBuildItem) {
+    private void configureDefaultPersistenceUnitEntities(ParsedPersistenceXmlDescriptor descriptor, CombinedIndexBuildItem indexBuildItem,
+            List<String> userManagedEntities) {
         IndexView index = indexBuildItem.getIndex();
         Collection<AnnotationInstance> annotations = index.getAnnotations(DotName.createSimple(Entity.class.getName()));
 
         for (AnnotationInstance annotation : annotations) {
             AnnotationTarget target = annotation.target();
             String targetName = target.asClass().name().toString();
 
-            if (isCustomJpaModel(targetName)) {
+            if (!userManagedEntities.contains(targetName)
+                    && !targetName.startsWith("org.keycloak")
+                    || targetName.startsWith("org.keycloak.testsuite")) {
```

Comment:
I think that this currently (without brackets) equals to:
```java
(!userManagedEntities.contains(targetName) && !targetName.startsWith("org.keycloak")) || targetName.startsWith("org.keycloak.testsuite")
```
Is that intended? I think it should rather be:
```java
!userManagedEntities.contains(targetName) && (!targetName.startsWith("org.keycloak") || targetName.startsWith("org.keycloak.testsuite"))
```
Am I correct?

In any case I think we should use brackets for better readability.

Reply:
Yeah, the outcome is the same but I can change it for better readbility.